quic/s2n-quic-platform/build.rs

```diff
@@ -16,17 +16,30 @@ fn main() -> Result<(), Error> {
         }
     }
 
+    let is_miri = std::env::var("CARGO_CFG_MIRI").is_ok();
```

Comment:
You can't use `cfg!(miri)` for this?

Reply:
The cfg isn't passed to build scripts themselves, since they run on a different "platform" than the code that they're compiling.